src/server.jsx

```diff
@@ -16,6 +16,7 @@ import nlLocale from '~/../locales/nl.json';
 import deLocale from '~/../locales/de.json';
 import enLocale from '~/../locales/en.json';
 import jaLocale from '~/../locales/ja.json';
+import ptBrLocale from '~/../locales/ptBR.json';
```

Comment:
@LeuAlmeida The portuguese brazilian locale code that you are using "ptBR" is wrong, in simple words, please rename the json file from "ptBR.json" to "pt_BR.json", this locale code is the used on the default plone core translations in portuguese brazilian.

- https://github.com/collective/plone.app.locales/blob/master/plone/app/locales/locales/pt_BR/LC_MESSAGES/plone.po

Reply:
Done!